rust/frontend/src/server.rs

```diff
@@ -151,59 +156,71 @@ impl FrontendServer {
     #[allow(dead_code)]
     pub async fn run(server: FrontendServer) {
         let circuit_breaker_config = server.config.circuit_breaker.clone();
+
+        let (docs_router, docs_api) =
+            OpenApiRouter::with_openapi(ApiDoc::openapi()).split_for_parts();
+
+        let docs_router = docs_router.merge(SwaggerUi::new("/docs").url("/openapi.json", docs_api));
+
         let app = Router::new()
             // `GET /` goes to `root`
-            .route("/api/v1/*any", get(v1_deprecation_notice).put(v1_deprecation_notice).patch(v1_deprecation_notice).delete(v1_deprecation_notice).head(v1_deprecation_notice).options(v1_deprecation_notice))
+            .route("/api/v1/{*any}", get(v1_deprecation_notice)
+                                     .put(v1_deprecation_notice)
+                                     .patch(v1_deprecation_notice)
+                                     .delete(v1_deprecation_notice)
+                                     .head(v1_deprecation_notice)
+                                     .options(v1_deprecation_notice))
             .route("/api/v2/healthcheck", get(healthcheck))
             .route("/api/v2/heartbeat", get(heartbeat))
             .route("/api/v2/pre-flight-checks", get(pre_flight_checks))
             .route("/api/v2/reset", post(reset))
             .route("/api/v2/version", get(version))
             .route("/api/v2/auth/identity", get(get_user_identity))
             .route("/api/v2/tenants", post(create_tenant))
-            .route("/api/v2/tenants/:tenant_name", get(get_tenant))
-            .route("/api/v2/tenants/:tenant_id/databases", get(list_databases).post(create_database))
-            .route("/api/v2/tenants/:tenant_id/databases/:name", get(get_database).delete(delete_database))
+            .route("/api/v2/tenants/{tenant_name}", get(get_tenant))
```

Comment:
OOC - why this change?

Reply:
Axum `0.8` changed the way path params are specified: https://github.com/tokio-rs/axum/releases/tag/axum-v0.8.0

> breaking: Upgrade matchit to 0.8, changing the path parameter syntax from /:single and /*many to /{single} and /{*many}; the old syntax produces a panic to avoid silent change in behavior (https://github.com/tokio-rs/axum/pull/2645)

Reply:
Ah but I misnamed it - should keep as tenant_id. Will fix that.

Reply:
No I think it was always tenant_name?

Reply:
Yeah looks like you're right. I just renamed everything to standardize on `tenant` and `database`, which matches our existing OpenAPI spec. 

The different variable names were causing issues with code generation, too.

Reply:
I don't think we should standardize on tenant and database tbh. It leads to a lot of bugs in confusion between name vs id for things like database. Wdyt? I guess maybe its fine since id is UUID and will fail to parse? But users get confused so the explicitness is nice.

IIRC @drewkim explicitly did it this way for clarity.

Reply:
I was basing it on our client experience, which call the params just `tenant` and `database`:

```
client = chromadb.HttpClient(
  ssl=True,
  host='api.trychroma.com',
  tenant='00000000-0000-0000-0000-000000000000',
  database='foo',
  headers={
      'x-chroma-token': 'YOUR_API_KEY'
  }
)
```

In practice - tenant is an id, but database is a name/slug. 

I think `database` and `tenant` look clean in the client libraries, and I think it helps to maintain the same variable name full-stack.  If we change the backend to refer to them as `database_name` and `tenant_id`, then somewhere becomes a boundary that converts them from `tenant -> tenant_id`. That can be the client library.  But, I think the

Reply:
✏️  Discussed offline